Hide authorization header in SARIF reports

de-jcup opened the pull request:
This PR
- closes #9

de-jcup committed:
Hide authorization header in SARIF reports

winzj reviewed (dismissed):
Looks good to me. 🚀

winzj reviewed (changes requested):
I'm sorry my first review was to fast, I saw that some actions failed.

de-jcup committed:
Fix Java 8 compile problem

- String.isBlank() is not available in Java 8, so replaced with
  String.isEmpty() - this is enough here and works with Java 8

de-jcup committed:
Merge branch 'sarif-reporting' into feature-hide-credentials-in-header

winzj reviewed (approved):
Looks good 👍
